go.sum

```diff
@@ -0,0 +1,4 @@
+gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405 h1:yhCVgyC4o1eVCa2tZl7eS0r+SDo693bJlVdllGtEeKM=
```

Comment:
I noticed that @niemeyer didn't have a go.sum.  It might be that the build or whatever else produces it.  I would not introduce this file, it seems like an unrelated change.

Reply:
done